examples/cstg/html/stylesheets/app.css

```diff
@@ -0,0 +1,98 @@
+body {
+  padding: 50px;
+  font: 14px "Lucida Grande", Helvetica, Arial, sans-serif;
+}
+
+a {
+  color: #00B7FF;
+}
+
+.button {
+  border-style: none;
+  cursor: pointer;
+  align-items: center;
+  height: 40px;
+  width: 401px;
+  text-align: center;
+  position: absolute;
+  letter-spacing: 0.28px;
+  box-sizing: border-box;
+  color: white;
+  font-family: 'Raleway', Helvetica, Arial, serif;
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 700;
+  text-transform: none;
+  text-indent: 0px;
```

Comment:
nit: we can drop the units when the value is 0

Reply:
Thanks! All CSS was copied from #4. Maybe then that is merged we can figure out a way to re-use the stylesheets across the example sites.

Reply:
ah I see. Is the plan for #4 to be merged first? Does it have to be?

Reply:
It doesn't matter the order they're merged in.